### PR TITLE
Changing phone number validation pattern to allow extensions

### PR DESCRIPTION
**Pull request by sachin-shrestha**
What type of PR is this?

Phone number validation pattern updated to allow extensions after the phone number.

### CODE REVIEW

**Comment by sachin-shrestha** [comment]
@jacobyavis @pjohnmeyer - looking for review/approval.

**Comment by sachin-shrestha** [comment]
@jacobyavis @pjohnmeyer - please review/approve. Thanks!

File: schema/types/Phone.schema.json [hunk 1]
```diff
@@ -10,9 +10,9 @@
       "$ref": "https://raw.githubusercontent.com/Open-Cap-Table-Coalition/Open-Cap-Format-OCF/main/schema/enums/PhoneType.schema.json"
     },
     "phone_number": {
-      "description": "A valid phone number string in ITU E.123 international notation (e.g. +123 123 456 7890)",
+      "description": "A valid phone number string in ITU E.123 international notation (e.g. +123 123 456 7890). An extension number, if applicable, should be separated by words ''extension'' or ''ext.'' after the phone number (e.g. +123 123 456 7890 ext. 100).",
       "type": "string",
-      "pattern": "^\\+\\d{1,3}\\s\\d{2,3}\\s\\d{2,3}\\s\\d{4}$"
+      "pattern": "^\\+\\d{1,3}\\s\\d{2,3}\\s\\d{2,3}\\s\\d{4}(?:\\s(ext.|extension)\\s\\d+)?$"
```
Comment (Sumukh) [inline schema/types/Phone.schema.json:15]: From the TWG call, the lookahead seems unnecessary. 

regexr.com/80kp6

`^\+\d{1,3}\s\d{2,3}\s\d{2,3}\s\d{4}(\s(ext.|extension)\s\d+)?$`

**Review by jacobyavis — APPROVED** [review]
Preparing to merge

**Pull request merged**